EncryptionBoxInput/EncryptionBoxInput.sol

```diff
@@ -0,0 +1,30 @@
+pragma ton-solidity >= 0.43.0;
+
+interface IEncryptionBoxInput {
+
+	function get(uint32 answerId, string prompt, string algorithm, TvmCell params) external returns (uint32 handle);
+    function getSupportedAlgorithms() external returns (string[] names);
```

Comment:
add function for box removing
function remove(uint32 handle) external returns (bool res);
to call https://github.com/tonlabs/TON-SDK/blob/master/docs/mod_crypto.md#remove_encryption_box